src/attribution/onINP.ts

```diff
@@ -239,10 +239,16 @@ const getIntersectingLoAFs = (
 
 const attributeINP = (metric: INPMetric): INPMetricWithAttribution => {
   const firstEntry = metric.entries[0];
+  const inpTime = firstEntry.startTime + metric.value;
```

Comment:
Nit: just call it `endTime` or something

Reply:
Updated